src/detect-engine-analyzer.c

```diff
@@ -861,6 +863,41 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_FLOWBITS: {
+                const DetectFlowbitsData *cd = (const DetectFlowbitsData *)smd->ctx;
+
+                jb_open_object(js, "flowbits");
+                jb_set_uint(js, "variables", cd->or_list_size);
+                switch (cd->cmd) {
+                    case DETECT_FLOWBITS_CMD_NOALERT:
+                        jb_set_string(js, "cmd", "noalert");
```

Comment:
To adhere more to our documentation, I think it makes sense we use "action" here, as that's what's used there.

Reply:
I'm sorry I don't quite understand what you mean. Are you saying that I should do this instead:
`jb_set_string(js, "cmd", "action");`?

Reply:
No, if we did that, `cmd` would always have value action in the output. I meant that instead of using `cmd` as the json key, we use `action`. So:
`jb_set_string(js, "action", ....)`

Reply:
oh, okay. Got it, thanks!